Expensify/App
# Completely remove getBetas and User_GetBetas

@luacmartins please review

HOLD ON MERGING UNTIL https://github.com/Expensify/App/pull/9934 IS DEPLOYED.

### Details
Completely remove User_GetBetas from E/App. We'll want to be sure to re-test this once 

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/213881

### Tests
1. Login via an expensifail.com account, make sure that you can see the #expensifail.com domain room.
1. Login via a vanilla Gmail account, make sure that you cannot see any default rooms (make sure you have not created a workspace, but have created a policy on OldDot that you are a member of)
